client/src/components/organisms/Header/Header.jsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import { Navbar, Nav, Button } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Could it have been achieved just by using CDN links

Reply:
I agree. My bad